charts/csi-cloudscale/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Release.Name }}-csi-cloudscale-controller-sa
+  namespace: {{ .Release.namespace }}
```

Comment:
Namespace should be in uppercase.

Reply:
In fact this looks to me like a copy paste error from our chart we were using locally sorry. As above Namespace is not part of the default `helm create <chartname>` template and is not needed, as it's defined as argument when installing the chart

Reply:
71a5e474a7474e430ac20013f8b7ec2c5fdf7e32